src/api/app/helpers/webui/package_helper.rb

```diff
@@ -87,7 +87,7 @@ def uploadable?(filename, architecture)
   def expand_diff?(filename, state)
     state != 'deleted' &&
       !filename.include?('/') &&
-      (filename == '_patchinfo' || filename.ends_with?('.spec') || filename.ends_with?('.changes'))
+      (filename == '_patchinfo' || filename.ends_with?('.spec', '.changes'))
```

Comment:
[nitpick] now that it is shorter, you could write the whole condition in a single line